<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAnnotationWidgetBase</class>
 <widget class="QWidget" name="QgsAnnotationWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QgsSymbolButton" name="mFrameStyleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mMapMarkerLabel_3">
     <property name="text">
      <string>Frame style</string>
     </property>
     <property name="buddy">
      <cstring>mMapMarkerButton</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="mMapPositionFixedCheckBox">
     <property name="text">
      <string>Fixed map position</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QgsCollapsibleGroupBox" name="mMarginsGroupBox">
     <property name="title">
      <string>Contents Margins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsDoubleSpinBox" name="mSpinTopMargin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Bottom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsDoubleSpinBox" name="mSpinLeftMargin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsDoubleSpinBox" name="mSpinRightMargin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QgsDoubleSpinBox" name="mSpinBottomMargin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QgsMapLayerComboBox" name="mLayerComboBox">
     <property name="toolTip">
      <string>Allows the annotation to be associated with a map layer. If set, the annotation will only be visible when the layer is visible.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mMapMarkerLabel">
     <property name="text">
      <string>Map marker</string>
     </property>
     <property name="buddy">
      <cstring>mMapMarkerButton</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsSymbolButton" name="mMapMarkerButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mMapMarkerLabel_2">
     <property name="text">
      <string>Linked layer</string>
     </property>
     <property name="buddy">
      <cstring>mMapMarkerButton</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mMapPositionFixedCheckBox</tabstop>
  <tabstop>mLayerComboBox</tabstop>
  <tabstop>mMapMarkerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
